code/datums/autolathe/engineering_vr.dm

```diff
@@ -0,0 +1,22 @@
+// Lets allow autolathes to build the basic tech 1 parts shall we?
```

Comment:
I vote no, because this lets you make infinite autolathes. This was a design choice not to let the autolathe make these.